app/assets/javascripts/components/overview/overview_handler.jsx

```diff
@@ -47,13 +50,29 @@ const Overview = createReactClass({
 
   componentDidMount() {
     ServerActions.fetch('timeline', this.props.course_id);
+    this.timeout = this.poll();
     return ServerActions.fetch('tags', this.props.course_id);
   },
 
+  componentWillUnmount() {
+    if (this.timeout) {
+      clearInterval(this.timeout);
+    }
+  },
+
   storeDidChange() {
     return this.setState(getState());
   },
 
+  poll() {
+    if (this.state.course.level === 'Editathon') {
```

Comment:
`course.level` doesn't look right.

Reply:
@ragesoss could you elaborate a little bit more please? Is the literal 'Editathon' I am comparing with, wrong?

Reply:
`course.type` is what I would expect, unless I'm missing something. Did you get this working locally?

Reply:
`course.type` contains values like `"ClassroomProgramCourse"` which I don't think is the case here. I checked out that the "BasicCourse", "Editathon" were under `course.level`. It worked for me locally, in the sense I could see the states being refreshed at intervals.

Reply:
course.type will be one of the values from `Course::COURSE_TYPES`.